src/clj/witan/send/utils.clj

```diff
@@ -352,3 +356,21 @@
 
 (defn int-ceil [n]
   (int (Math/ceil n)))
+
+(defn reset-log []
+  (def log (atom []))
```

Comment:
you need to define log *outside* of the defn. You should never have a `def` inside a defn.

Reply:
I'd actually move the logging stuff to a witan.send.report ns and rename `log-info` to `append`